gen2/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Clang = "40e3b903-d033-50b4-a0cc-940c62c95e31"
```

Comment:
Hmmm, I think we can safely keep both `libgifextra_jll` and `Giflib_jll` in the same `gen/Project.toml` __and `gen/Manifest.toml`__ file and use one single `generater.jl` to generate both wrapped modules.